strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitChangesDetector.java

```diff
@@ -84,6 +102,15 @@ private File findRepoRoot(File currentDir) {
                 test, previous, head)).collect(Collectors.toList());
     }
 
+    public void appendLocalFiles(Set<File> files, Set<String> localFiles) {
+        files.addAll(
```

Comment:
Maybe instead of manipulating `files` set we could simply return the filtered collection and decide on the caller side what to do with the result?